.github/workflows/add-qa-link-to-issues.yml

```
@@ -19,4 +19,5 @@ jobs:
       - uses: mshick/add-pr-comment@v2
         with:
           message: |
-            View your branch deployment here: https://mi6.github.io/ic-ui-kit/branches/${{ steps.extract_branch.outputs.branch }}
\ No newline at end of file
+            View your branch deployment here: https://mi6.github.io/ic-ui-kit/branches/${{ steps.extract_branch.outputs.branch }}
```

Update comment to match new location
```suggestion
            View your branch deployment here: https://mi6.github.io/ic-ui-kit/branches/${{ steps.extract_branch.outputs.branch }}/web-components
```